changedetectionio/model/Watch.py

```diff
@@ -196,6 +198,39 @@ def save_history_text(self, contents, timestamp):
         # @todo bump static cache of the last timestamp so we dont need to examine the file to set a proper ''viewed'' status
         return snapshot_fname
 
+    # Save previous text snapshot for diffing - used for calculating additions and deletions
+    def save_previous_text(self, contents):
+        import logging
+
+        output_path = os.path.join(self.__datastore_path, self['uuid'])
+
+        # Incase the operator deleted it, check and create.
+        if not os.path.isdir(output_path):
```

Comment:
You can remove this and call https://github.com/dgtlmoon/changedetection.io/blob/57f604dff1580ec49e58469edf616b83f0af9758/changedetectionio/model/Watch.py#L86

Reply:
Completed in https://github.com/dgtlmoon/changedetection.io/pull/749/commits/c2c8bb4de86af5209f36fee9e5b0902b74556bc9